virttest/utils_misc.py

```diff
@@ -2675,6 +2675,17 @@ def get_winutils_vol(session, label="WIN_UTILS"):
     return get_win_disk_vol(session, condition="VolumeName='%s'" % label)
 
 
+def set_winutils_letter(session, cmd, params):
```

Comment:
where used param argument? and winutils volume letter is not always 'X', so I think you need to rework this function